zebra-state/src/service/finalized_state/zebra_db/shielded.rs

```diff
@@ -100,6 +100,17 @@ impl FinalizedState {
         self.db.zs_get(sprout_anchors, sprout_anchor)
     }
 
+    /// Returns the Sprout note commitment tree matching the given block height.
+    #[allow(dead_code)]
+    pub fn sprout_note_commitment_tree_by_height(
+        &self,
+        height: &Height,
+    ) -> Option<sprout::tree::NoteCommitmentTree> {
+        let sprout_trees = self.db.cf_handle("sprout_note_commitment_tree").unwrap();
+
+        self.db.zs_get(sprout_trees, height)
+    }
```

Comment:
Disk Use follow-up: this method doesn't make any sense if we're still deleting sprout trees.